SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -12,16 +12,8 @@ Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setti
 > To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
 
 ## Setting up the project structure
-1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `BuildGDK.bat`. This requires authorization with your SpatialOS account.
-1. In File Explorer, make a copy of the SpatialOS Unreal GDK Starter Project repository you cloned. This copy will become the root directory of the Unreal project you're porting over, so paste it where you want this project to be.
-1. Rename the copied directory so it's relevant to your Unreal game. We'll refer to this directory as `<ProjectRoot>`.
-1. Within `<ProjectRoot>`, replace the `Game` directory with the equivalent directory for your Unreal game (we'll refer to this as `<GameRoot>`).
+1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `Setup.bat`. This requires authorization with your SpatialOS account.
```

Comment:
This only applies if you have cloned the GDK into the plugins folder of your game. it does not seem to be mentioned above

Reply:
This doc states that you should have followed the getting started guide, up to the cloned section.